Makefile.doc

```diff
@@ -0,0 +1,16 @@
+COQDOCJS_DIR ?= coqdocjs
+EXTRA_DIR = $(COQDOCJS_DIR)/extra
+COQDOCFLAGS ?= \
+  --external 'http://ssr2.msr-inria.inria.fr/doc/ssreflect-1.5/' Ssreflect \
+  --external 'http://ssr2.msr-inria.inria.fr/doc/mathcomp-1.5/' MathComp \
```

Comment:
These links are deprecated, and I am not sure we should single out mathcomp. Maybe introducing another variable such as `COQDOCFLAGS_EXTRA` that gets expanded here is and is empty by default is preferable. This would also allow passing `-s` (which I usually do)